carmajava/launch/guidance.launch

```diff
@@ -155,8 +154,5 @@
     <arg name="route_file_folder" value="$(arg route_file_folder)"/>
   </include>
 
-  <!-- Health Monitor -->
   <remap from="~required_drivers" to="/required_drivers"/>
```

Comment:
You need to move this line to inside the health monitor node tag in carma_src.launch